.rubocop.yml

```diff
@@ -137,10 +137,6 @@ Style/FloatDivision:
 Style/FrozenStringLiteralComment:
   Enabled: false
 
-# use 1.8 style hashes
-Style/HashSyntax:
-  EnforcedStyle: hash_rockets
-
```

Comment:
Alternatively, we could reduce the impact of this change by instead allowing both the old way and the new way:

```suggestion
# Allow use of 1.8 and 1.9 style hashes, but not both in the same hash
Style/HashSyntax:
  EnforcedStyle: no_mixed_keys
```